### PR TITLE
mariadb@10.10 10.10.6

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
```
  -- Performing Test HAVE_SYSTEM_LIBFMT
  -- Performing Test HAVE_SYSTEM_LIBFMT - Failed
  CMake Error at cmake/libfmt.cmake:45 (MESSAGE):
    system libfmt library is not found or unusable
  Call Stack (most recent call first):
    CMakeLists.txt:395 (CHECK_LIBFMT)
```

**Comment by ZhongRuoyu** [comment]
To be safe, I suggest waiting for upstream maintainers' feedback on the patch first.

**Comment by chenrui333** [comment]
Nice work!

- relates to https://github.com/MariaDB/server/pull/2732

File: Formula/m/mariadb@10.10.rb
Comment (samford) [inline Formula/m/mariadb@10.10.rb:4]:
```suggestion
  url "https://archive.mariadb.org/mariadb-10.10.6/source/mariadb-10.10.6.tar.gz"
```

Comment (ZhongRuoyu) [reply Formula/m/mariadb@10.10.rb]: Applied suggestion.

File: Formula/m/mariadb@10.10.rb
Comment (ZhongRuoyu) [inline Formula/m/mariadb@10.10.rb:6]: Corrected the license; see https://github.com/MariaDB/server/blob/mariadb-10.10.6/COPYING#L299

**Review by iMichka — APPROVED** [review]

**Pull request merged**
